### PR TITLE
Retrieving blazons and flags from DBpedia

### PR DESCRIPTION
**Pull request by davidbgk**
Retrieve the flag or blazon file name and the dbpedia URL.

### CODE REVIEW

File: france.py [hunk 1]
```diff
@@ -537,20 +537,22 @@ def fetch_missing_data_from_dbpedia(db, filename):
...
+            'wikipedia': {'$exists': True, '$ne': None},
+            '$or': [
+                {'population': None},
+                {'population': {'$exists': False}},
+                {'area': None},
+                {'area': {'$exists': False}},
+            ]
+            }, no_cursor_timeout=True):
+
+        dbpedia = DBPedia(zone['wikipedia'])
```
Comment (noirbizarre) [inline france.py]: No more failsafe processing ? (try...catch) ? DBPedia is failsafe now ?

**Comment by noirbizarre** [comment]
r+wc

File: france.py (hunk 1 above)
Comment (davidbgk) [inline france.py]: I ran the script without any error, so I guess it's OK?

Comment (davidbgk) [inline france.py]: Actually I added the `.` in the regexp to turn it safe ;)

**Pull request merged**